src/routes/signup/+page.server.ts

```diff
@@ -41,9 +43,11 @@ export const actions: Actions = {
 		}>({
 			name: Joi.string().min(3).max(31).optional(),
 			username: Joi.string().min(3).max(31),
-			email: Joi.string().email().optional(),
+			email: Joi.string().email().required(),
```

Comment:
I'm not yet sure if email is necessary. Not at the time, at least.

Reply:
It's mandatory on UI tho. So we either make it optional on UI or mandatory here. whichever you prefer. I think optional makes sense because we already have username. But in future if we introduce notification feature then we are gonna need email. 

![image](https://github.com/user-attachments/assets/bada6278-8876-437d-9e1c-fab96a5e91a5)

Reply:
Ok, so maybe let's make it optional for now.